Sources/ScrollingTabView.swift

```diff
@@ -31,11 +31,13 @@ public let ScrollingTabTitleCell = "TabCell"
  * View that contains the top set of tabs and their containing collection view.
  */
 public class ScrollingTabView: UIView {
-    
+
     public enum TabSizing {
         case fitViewFrameWidth
         case fixedSize(CGFloat)
         case sizeToContent
+        ///Takes on the attributes of fitViewFrameWidth until the content is too large and then it takes on the attributes of sizeToContent
+        case flexibleWidth
```

Comment:
can we call this sizeToWidthWithOverflow or something similar. Would likely be more descriptive.